### PR TITLE
Pin libnetcdf

### PR DESCRIPTION
**Pull request by ocefpaf**
@jgomezdans can you review this to see if everything from your PR is here?

@gillins can you review and merge after @jgomezdans gives the :+1: ?

---
Edit: This PR supersedes #67

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
             --with-jpeg=$PREFIX \
             --with-libtiff=$PREFIX \
             --with-png=$PREFIX \
+            --with-sqlite3=$PREFIX \
+            --with-spatialite=$PREFIX \
+            --with-curl \
```
Comment (ocefpaf) [inline recipe/build.sh]: @jgomezdans just curious: why curl here?

Comment (jgomezdans) [inline recipe/build.sh]: I just had it on a previous version of my build file, so left it over. In all my build, curl is being identified automatically, so this isn't probably needed.

**Comment by jgomezdans** [comment]
Looks good to me, but feel free to get rid of the `--with-curl` in `build.sh` if you think it will cause problems. Otherwise, go ahead and commit ;-)

**Comment by ocefpaf** [comment]
Better explicit than implicit. Let's leave it there. I'll merge once I get
back home if someone does not beat me to it. Thanks again for the
contribution José!
Em 04/06/2016 09:08, "José Gómez-Dans" <notifications@github.com> escreveu:

**Pull request merged**